package-lock.json

```diff
@@ -15629,9 +15629,9 @@
       "dev": true
     },
     "vue-loader": {
-      "version": "15.7.1",
-      "resolved": "https://registry.npmjs.org/vue-loader/-/vue-loader-15.7.1.tgz",
-      "integrity": "sha512-fwIKtA23Pl/rqfYP5TSGK7gkEuLhoTvRYW+TU7ER3q9GpNLt/PjG5NLv3XHRDiTg7OPM1JcckBgds+VnAc+HbA==",
```

Comment:
Do you think this change to the lockfile can be reverted? The lockfile usually should only change when dependencies are changed (added, removed, updated).